openedx/features/caliper_tracking/tests/expected/book.json

```diff
@@ -0,0 +1,43 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "NavigatedTo",
+    "actor": {
+        "id": "http://localhost:18000/u/staff",
+        "name": "staff",
+        "type": "Person"
+    },
+    "eventTime": "2018-10-19T07:49:07.392Z",
+    "extensions": {
+        "extra_fields": {
+            "accept_language": "en-US,en;q=0.9",
+            "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+            "course_id": "course-v1:edX+DemoX+Demo_Course",
+            "event_source": "browser",
+            "event_type": "book",
+            "host": "8433fad63981",
+            "ip": "172.18.0.1",
+            "org_id": "edX",
+            "page": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/pdfbook/0/?viewer=true&file=/assets/courseware/v1/55969d87f0409b18ed8d979465e80f1a/asset-v1:edX+DemoX+Demo_Course+type@asset+block/gazatte_11th_2018.pdf#zoom=page-fit&disableRange=true",
+            "path": "/event",
+            "session": "1e2bcef58672f0195f63d9e2f0736a7c",
+            "user_id": 8
+        }
+    },
+    "id": "urn:uuid:4a725288-39af-4735-a3b4-21e2c002ac38",
+    "object": {
+        "extensions": {
+            "chapter": "/assets/courseware/v1/55969d87f0409b18ed8d979465e80f1a/asset-v1:edX+DemoX+Demo_Course+type@asset+block/gazatte_11th_2018.pdf",
+            "name": "textbook.pdf.page.loaded",
```

Comment:
in event 'textbook.pdf.zoom.menu.changed ' you exclude name field from extensions and here you leave it as it is? Any explanations ?

Reply:
Added in that event as well.